athaliana_example.py

```diff
@@ -0,0 +1,23 @@
+"""
+A. thaliana msprime example
+"""
+import msprime
+
+import stdpopsim
+from stdpopsim import arabidopsis_thaliana
+
+chrom = arabidopsis_thaliana.genome.chromosomes["chr5"]
+recomb_map = chrom.recombination_map()
+
+model = arabidopsis_thaliana.Durvasula2017MSMC()
+model.debug()
+
+samples = [msprime.Sample(population=0, time=0),msprime.Sample(population=0,
```

Comment:
Usually the best way to break lines like this is to use hanging indents:
```python
samples = [
    msprime.Sample(population=0, time=0), msprime.Sample(population=0, time=0)]
```
But, style is subjective...